cache/blobs.go

```diff
@@ -169,7 +193,7 @@ func computeBlobChain(ctx context.Context, sr *immutableRef, createIfNeeded bool
 					}
 				}
 				if enableOverlay {
-					computed, ok, err := sr.tryComputeOverlayBlob(ctx, lower, upper, mediaType, sr.ID(), compressorFunc)
+					computed, ok, err := sr.tryComputeOverlayBlob(ctx, lower, upper, mediaType, sr.ID(), compressorFunc, sourceDateEpoch)
 					if !ok || err != nil {
```

Comment:
- FWIW, this file is also updated in https://github.com/moby/buildkit/pull/3867 - does that PR need any changes for this to work with `overlaybd` ? /cc @HileQAQ

Reply:
Probably yes, but we can defer that, as `overlaybd` will stay optional.